projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.ts

```diff
@@ -71,6 +89,12 @@ export class TableBasicComponent implements OnInit, OnDestroy {
         this.onDestroy$.complete();
     }
 
+    ngAfterViewInit() {
+        if (this.tableService.hasRowspanRows) {
+            this.setHoverForRowspan();
```

Comment:
If the table has row span rows - need to do a subscription to apply hover CSS class